src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -516,15 +516,18 @@ public function toggleStatusAction($customerId)
 
             $this->getCommandBus()->handle($editCustomerCommand);
 
-            $this->addFlash(
-                'success',
-                $this->trans('The status has been successfully updated.', 'Admin.Notifications.Success')
-            );
+            $response = [
+                'status' => true,
+                'message' => $this->trans('The status has been successfully updated.', 'Admin.Notifications.Success'),
+            ];
         } catch (CustomerException $e) {
-            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+            $response = [
+                'status' => false,
+                'message' => $this->getErrorMessageForException($e, $this->getErrorMessages($e)),
+            ];
         }
 
-        return $this->redirectToRoute('admin_customers_index');
+        return $this->json($response);
```

Comment:
Please update PHPDoc : it's not anymore a `RedirectResponse`

Reply:
And it's a BC break since you change the contract and behaviour of the "API", no more redirection but JSON formatted response